src/editor.js

```diff
@@ -264,9 +275,77 @@ const ContentPanel = ({
         codeMirrorElement is a promise resolved by codemirror(), hence the
         until directive. Once resolved, content can be empty.
       -->
+    ${TemplateLightbox({isTemplateLightboxDisplayed})}
     ${until(codeMirrorElement || Textarea({content}))}
   </div>
 `;
+const dispatchLoadHtmlFile = redispatchAs(g('load-html'));
+
+const TemplateLightbox = ({isTemplateLightboxDisplayed}) => html`
+  <div
+    class="${n('template-lightbox')}"
+    ?hidden=${!isTemplateLightboxDisplayed}
+  >
+    ${getNamesOfTemplates()}
+  </div>
+`;
+
+function getNamesOfTemplates() {
+  return html`
+    <div class="${g('flex-center')}">
```

Comment:
Get the `templates` here and iterate them. They all have a `preview` prop as set in my snippet above. Depending on their `type`, they either render a `<video>` or an `<img>`. (Again, idea is not to have anything hardcoded.)